get_all_manifests.sh

```diff
@@ -15,9 +15,10 @@ declare -A COMPONENT_MANIFESTS=(
     ["ray"]="opendatahub-io:distributed-workloads:main:ray:ray"
     ["data-science-pipelines-operator"]="opendatahub-io:data-science-pipelines-operator:main:config:data-science-pipelines-operator"
     ["odh-dashboard"]="opendatahub-io:odh-dashboard:incubation:manifests:odh-dashboard"
-    ["kf-notebook-controller"]="opendatahub-io:kubeflow:v1.7-branch:components/notebook-controller/config:odh-notebook-controller/kf-notebook-controller"
-    ["odh-notebook-controller"]="opendatahub-io:kubeflow:v1.7-branch:components/odh-notebook-controller/config:odh-notebook-controller/odh-notebook-controller"
+    ["kf-notebook-controller"]="cam-garrison:kubeflow:add-sm-overlay:components/notebook-controller/config:odh-notebook-controller/kf-notebook-controller"
```

Comment:
?

Reply:
Please take a look at merge criteria of the PR description